packages/esm-patient-tests-app/src/edit-test-results/modal/edit-lab-results.modal.tsx

```diff
@@ -0,0 +1,118 @@
+import { Button, ModalBody, ModalFooter, ModalHeader, RadioButton, RadioButtonGroup } from '@carbon/react';
+import { launchWorkspace, useSession } from '@openmrs/esm-framework';
+import { type Order } from '@openmrs/esm-patient-common-lib';
+import classNames from 'classnames';
+import lowerCase from 'lodash-es/lowerCase';
+import upperCase from 'lodash-es/upperCase';
```

Comment:
@its-kios09 there a number of use cases for these utilities in this file and this shouldn't be the case considering we depend on strings from translations. 

There was also a comment in a previous review about the same thing https://github.com/openmrs/openmrs-esm-patient-chart/pull/2334#discussion_r2050824979